install-raspbian.sh

```diff
@@ -269,6 +269,16 @@ EOF
 
 fi
 
+info "### Enable Nodejs GPIO access - please reboot"
```

Comment:
Maybe optional via request?

Reply:
The new overlay in /boot/config.txt will only activate at reboot. I did not find any way to activate this without reboot. And without this overlay, there is stability issues it seems.

Of course the reboot at this point is optional and not using the remote buzzer feature, the reboot is not required at all. 

I would not add a prompt here as right at the end of the script, anyways the user is prompted for a possible reboot.